### PR TITLE
Add CM autolathe

### LINKED ISSUE #852: Autolathe sprite
https://cm-ss13.com/wiki/File:Autolathe.png

### LINKED ISSUE #1983: Disable non-cm lathe recipes

### PR DESCRIPTION
**Pull request by DrSmugleaf**
## About the PR
Fixes #852
Fixes #1983
Includes the changes in https://github.com/space-wizards/space-station-14/pull/27842

## Media
Content Client_BNpONNDgtg

- [x] I have added screenshots/videos to this PR showcasing its changes ingame, **or** this PR does not require an ingame showcase

**Changelog**
:cl:
- add: Added the autolathe with cm specific recipes.

### CODE REVIEW

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 1]
```diff
@@ -0,0 +1,114 @@
...
+        layer:
+        - MachineLayer
+  - type: Lathe
+    idleState: autolathe
+    runningState: autolathe_n
+    staticRecipes:
+    - CMBucket
+#    - Janitorial bucket (10000 metal)
+#    - Flashlight (62 metal, 25 glass)
+#    - Extinguisher (112 metal)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:37]: this exists

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 2]
```diff
@@ -0,0 +1,114 @@
...
+    - CMTRayScanner
+    - CMWelderSmall
+    - CMScrewdriver
+    - CMWirecutter
+    - CMWrench
+#    - Mop (137 metal)
+#    - Radio headset (93 metal)
+#    - Shortwave radio (93 metal, 31 glass)
+#    - Welding mask (3750 metal, 1250 glass)
+#    - Steel sheets (3750 metal)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:49]: remove this

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 3]
```diff
@@ -0,0 +1,114 @@
...
+#    - Glass sheets (3750 glass)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:50]: remove this

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 4]
```diff
@@ -0,0 +1,114 @@
...
+#    - Reinforced glass sheets (1875 metal, 3750 glass)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:51]: I think this exists

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 5]
```diff
@@ -0,0 +1,114 @@
...
+#    - Metal rods (2343 metal)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:52]: this exists

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml [hunk 6]
```diff
@@ -0,0 +1,114 @@
...
+#    - Reinforced table parts (18750 metal)
+#    - Rack parts (4687 metal)
+    - CMScalpel
+    - CMCircularSaw
+    - CMSurgicalDrill
+    - CMRetractor
+    - CMCautery
+    - CMHemostat
+#    - Glass beaker (625 glass)
+#    - Large glass beaker (6250 glass)
```
Comment (QuietlyWhisper) [inline Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml:69]: these both exist

**Review by QuietlyWhisper — CHANGES_REQUESTED** [review]

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 1 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 2 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 3 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 4 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 5 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

File: Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml (hunk 6 above)
Comment (DrSmugleaf) [reply Resources/Prototypes/_CM14/Entities/Structures/Machines/Lathe/lathe.yml]: Done in [`6c04fa0` (#2085)](https://github.com/CM-14/CM-14/pull/2085/commits/6c04fa01a492f738e781657d72b8696c7c3582e4)

**Pull request merged**